catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -127,6 +127,18 @@ public UpdateRequest process(UpdateRequest input)
               validatorName);
           return null;
         } else {
+          if (validationErrorsExist) {
+            INGEST_LOGGER.debug(
```

Comment:
If we could pull the filename/title and log that here along with the content of the getErrors and getWarnings that would be very helpful.

Reply:
fixed

Reply:
fixed